API.md

```diff
@@ -375,11 +375,16 @@ Replaces a document and returns the count of modified documents where:
  - `options` - an options object passed to MongoDB's native
    [`replaceOne`](https://docs.mongodb.com/manual/reference/method/db.collection.replaceOne/)
    method.
- - `callback` - the callback method using the signature `function (err, count)`
+ - `callback` - the callback method using the signature `function (err, count, result)`
    where:
     - `err` - if the query failed, the error reason, otherwise `null`.
     - `count` - if the query succeeded, a number indicating how many documents
       were modified.
+    - `result` - if the query succeeded, the raw result document returned
```

Comment:
Let's simplify the description for these to just:

`if the query succeeded, the raw result document returned by MongoDB's native driver.`

This way we're not responsible to document what's not in our control (if they add/remove things from the results document for example).